mitosheet/src/components/taskpanes/Graph/GraphSidebar.tsx

```diff
@@ -149,18 +148,22 @@ const GraphSidebar = (props: {
         size.
     */
     const getGraphAsync = async () => {
-        const boundingRect: DOMRect | undefined = document.getElementById('graph-div')?.getBoundingClientRect();
+        // The reason that we use the mito-main-sheet-div instead of the graph-div is because the size of the graph div
```

Comment:
but isn't the `Mito-main-sheet-div` small at this point? Namely, isn't it 0 width (or is that just the grid we shrink down...). 

Ohhh... it's the main container. I see what you mean then. Ok... I am ok with this!